### PR TITLE
[skip ci] Multiple VLAN cleanup fix for nightly

### PR DESCRIPTION
**Pull request by rajanashok**
Updated Create a Simple VC Cluster to return the vm names so to pass it to the Nimbus Cleanup during Suite teardown.

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 Documentation  Test 5-12 - Multiple VLAN
 Resource  ../../resources/Util.robot
-Suite Setup  Create a Simple VC Cluster  multi-vlan-1  cls
-Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
 Test Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Test1
+    ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}=  Create a Simple VC Cluster  multi-vlan-1  cls
```
Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]: Sure, I like the idea on the thin wrapper, but can I make it return values and use it from the Suite Setup?

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]: no thus, you need the extra keyword that thinly wraps it.

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]: Sorry, I am not following you. How does the extra keyword helps with the return values($list) so I can pass it to the Nimbus Cleanup.

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]:
```
*** Settings ***
Documentation  Test 5-12 - Multiple VLAN
Resource  ../../resources/Util.robot
Suite Setup  Multiple VLAN Setup
Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
Test Teardown  Cleanup VIC Appliance On Test Server

*** Keywords ***
Multiple VLAN Setup
    ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}=  Create a Simple VC Cluster  multi-vlan-1  cls
    Set Global Variable  @{list}  ${esx1}  ${esx2}  ${esx3}  ${vc}

*** Test Cases ***
Test1
```

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]: Ahh Got it! thanks.

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot]: Pushed the updated commit.

File: tests/resources/Nimbus-Util.robot [hunk 2]
```diff
@@ -306,7 +306,7 @@ Create a Simple VC Cluster
     Set Environment Variable  TEST_DATASTORE  datastore1
     Set Environment Variable  TEST_RESOURCE  ${cluster}
     Set Environment Variable  TEST_TIMEOUT  30m
-    [Return]  ${vc-ip}
+    [Return]  ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}
```
Comment (mhagen-vmware) [inline tests/resources/Nimbus-Util.robot]: I think other tests use this keyword, does this break them?

Comment (rajanashok) [inline tests/resources/Nimbus-Util.robot]: Yeah I thought about it, looks like other places where this is being called does not return any values. I will make sure if that is the case.

Comment (rajanashok) [inline tests/resources/Nimbus-Util.robot]: All the other files has a reference to the keyword without assigning any values from the function return. So we are not breaking the method contract. I am guessing we should be good?

Create a Simple VC Cluster  datacenter  cls

**Comment by mhagen-vmware** [comment]
lgtm

**Pull request merged**